### PR TITLE
importer dimensions plugged in

### LINKED ISSUE #674: Importer allows for too many dimensions when setting the unit of measurement
Issue to plug in to MoBi, description in Core:

Open-Systems-Pharmacology/OSPSuite.Core#1426

### PR DESCRIPTION
**Pull request by georgeDaskalakis**
closes #674

### CODE REVIEW

File: src/MoBi.Presentation/MoBi.Presentation.csproj [hunk 1]
```diff
@@ -28,6 +28,9 @@
     <PackageReference Include="Northwoods.GoWin" Version="5.2.0" />
     <PackageReference Include="OSPSuite.TeXReporting" Version="3.0.0.4" />
     <PackageReference Include="OSPSuite.Utility" Version="4.0.0.4" />
+    <PackageReference Include="OSPSuite.Presentation.Importer" Version="10.0.92" />
+    <PackageReference Include="OSPSuite.TeXReporting" Version="3.0.0.4" />
+    <PackageReference Include="OSPSuite.Utility" Version="4.0.0.4" />
```
Comment (msevestre) [inline src/MoBi.Presentation/MoBi.Presentation.csproj:33]: we have multiple of the same again. WTF

File: src/MoBi.Engine/MoBi.Engine.csproj [hunk 2]
```diff
@@ -32,7 +32,7 @@
   </ItemGroup>
 
   <ItemGroup>
-     <PackageReference Include="OSPSuite.Core" Version="11.0.65" />
+	  <PackageReference Include="OSPSuite.Core" Version="11.0.65" />
```
Comment (msevestre) [inline src/MoBi.Engine/MoBi.Engine.csproj:35]: indent

**Review by msevestre — CHANGES_REQUESTED** [review]
Look at the csproj closely . Indent issue and duplicate entries

File: src/MoBi.Presentation/MoBi.Presentation.csproj (hunk 1 above)
Comment (georgeDaskalakis) [reply src/MoBi.Presentation/MoBi.Presentation.csproj:33]: this is weird....haven't seen this one before...

File: src/MoBi.Engine/MoBi.Engine.csproj (hunk 2 above)
Comment (georgeDaskalakis) [reply src/MoBi.Engine/MoBi.Engine.csproj:35]: corrected the first, for the second I have no clue - locally it seems fine...

**Comment by msevestre** [comment]
Ok....

**Review by msevestre — APPROVED** [review]

File: src/MoBi.Presentation/MoBi.Presentation.csproj [hunk 3]
```diff
@@ -26,13 +26,11 @@
 
   <ItemGroup>
     <PackageReference Include="Northwoods.GoWin" Version="5.2.0" />
+    <PackageReference Include="OSPSuite.Presentation.Importer" Version="10.0.92" />
```
Comment (georgeDaskalakis) [inline src/MoBi.Presentation/MoBi.Presentation.csproj:29]: @msevestre  I ve been through the project files thoroughly to remove the duplicates and I find this line a bit weird...What is OSPSuite.Presentation.Importer and why its version is 10.0.92? Doesn't it come from Core? You think it is something deprecated?

Comment (msevestre) [reply src/MoBi.Presentation/MoBi.Presentation.csproj:29]: yes this does not exist anymore. This should be removed

**Pull request merged**